PokeAlarm/Events/QuestEvent.py

```diff
@@ -19,10 +21,9 @@ def __init__(self, data):
         # Identification
         self.stop_id = data['pokestop_id']
         self.stop_name = check_for_none(
-            str, data.get('pokestop_name') or data.get('name'),
-            Unknown.REGULAR)
+            str, data.get('pokestop_name', data.get('name')), Unknown.REGULAR)
         self.stop_image = check_for_none(
-            str, data.get('pokestop_url') or data.get('url'), Unknown.REGULAR)
+            str, data.get('pokestop_url', data.get('url')), Unknown.REGULAR)
```

Comment:
This looks wrong - what are you trying to do?

Reply:
see #691 - sometimes `or` evaluates things as falsey, this seems like the best remedy